basis/ui/gadgets/scrollers/scrollers.factor

```diff
@@ -96,10 +96,12 @@ M: rect update-scroller swap (scroll>rect) ;
 M: f update-scroller drop (update-scroller) ;
 
 M: scroller layout*
-    [ call-next-method ] [
-        dup follows>>
-        [ update-scroller ] [ >>follows drop ] 2bi
-    ] bi ;
+    {
+      [ call-next-method ]
```

Comment:
We've been using 4 space indentation (not that it matters too much) but trying to be consistent.